kinta-cli/src/main/resources/settings.gradle.kts

```diff
@@ -1,5 +1 @@
-rootProject.name="kintaSrc"
-
-if (File("../../kinta").isDirectory()) {
-    includeBuild("../../kinta")
-}
```

Comment:
This was useful during development but I guess we're aiming for stable versions now ?